GDJS/Runtime/events-tools/objecttools.ts

```diff
@@ -585,6 +585,13 @@ namespace gdjs {
         return count;
       };
 
+      export const setIncludedInParentCollisionMask = (
+        child: gdjs.RuntimeObject,
```

Comment:
You're sure you don't want to pass lists of objects here? So that all the passed instances are changed?